posts/solving-parallel-letter-frequency-elixir-part-1-unicode-match-regex.md

```diff
@@ -1,6 +1,8 @@
-# "Solving Parallel Letter Frequency in Elixir - Part I: Unicode Matching in Elixir"
+# 1. What Parallel Letter Frequency can teach you about Unicode matching in Elixir
 
-[Parallel Letter Frequency](https://exercism.io/tracks/elixir/exercises/parallel-letter-frequency) is a medium difficulty problem on [Exercism's Elixir Track](https://exercism.io/tracks/elixir) that I found really interesting to solve. You need to write a function that calculates the frequency of letters in a list of strings containing _non-English characters_ with a user-controlled number of workers that do the calculation _concurrently_:
+Exercises on Exercism are small, synthetic, and often seemingly trivial. It’s easy to imagine that experienced practitioners would have nothing to learn from them. However, solving these synthetic problems can push you to learn and apply parts of your language that you may not have explored. This new learning can lead you to solve real world problems more efficiently or in a more expressive way.
+
+[Parallel Letter Frequency](https://exercism.io/tracks/elixir/exercises/parallel-letter-frequency) is a medium difficulty exercise on [Exercism's Elixir Track](https://exercism.io/tracks/elixir). It asks you to write a function that calculates the frequency of letters in a list of strings, and to do so concurrently. This unpacks a surprising number of interesting lessons.
```

Comment:
We need a transition here to introduce one of those lessons as Unicode, and then a transition from there to explain why the reader is looking at a code example :)